OSC/ondemand
# Script#edit support adding/removing accounts

`script#edit` support for adding & removing accounts (`auto_accounts`) through the UI.   It also fixes the test cases around accounts.

to test
* make a script
* edit that script
* save or remove `Account`. 
* the resulting yml should have account options in it if you save and remove them if you remove it.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1204658906644267) by [Unito](https://www.unito.io)
